docs/docs/10.2-form-input-binding-sugar.md

```diff
@@ -97,16 +97,16 @@ As you can see, `ReactLink` objects are very simple objects that just have a `va
 ### ReactLink Without valueLink
 
 ```javascript
-var LinkedStateMixin = require('react-addons-linked-state-mixin');
+const LinkedStateMixin = require('react-addons-linked-state-mixin');
 
-var WithoutLink = React.createClass({
+const WithoutLink = React.createClass({
   mixins: [LinkedStateMixin],
-  getInitialState: function() {
+  getInitialState() {
     return {message: 'Hello!'};
   },
-  render: function() {
-    var valueLink = this.linkState('message');
-    var handleChange = function(e) {
+  render() {
+    const valueLink = this.linkState('message');
+    const handleChange = (e) => {
```

Comment:
We should not be using experimental syntax such as class properties in the documentation. In general I don’t think it’s necessary to ES6-ify a `createClass` example.